app/models/form.rb

```diff
@@ -114,6 +118,13 @@ def metrics_data
     nil
   end
 
+  def path_to_form
+    return Rails.application.routes.url_helpers.live_form_path(id) if is_live?
```

Comment:
Yeh I wasn't very sure about this - but wanted to share this code somewhere as it's used in 2 places now and seems like it could be used in more. Any more sensible place to share the code? In a helper class?

Reply:
Nvm just saw your other comment